test/scala/controllers/v1/BusinessControllerSpec.scala

```diff
@@ -90,13 +91,14 @@ class BusinessControllerSpec extends FreeSpec with Matchers with MockFactory wit
         contentType(response) shouldBe Some(JSON)
         contentAsJson(response).as[Seq[Business]] should contain theSameElementsInOrderAs
           Seq(SampleBusinessWithAllFields.secured, SampleBusinessWithAllFields1.secured)
```

Comment:
use businesses here rather than repeating the definition?
(this helps the next assertion on businesses.length to make more sense - otherwise the assertions seem a little disjoint)